library/src/morpheus-master/sass/modules/_toolmenu.scss

```diff
@@ -330,7 +330,7 @@ nav#subSites{
 						box-shadow: $tool-menu-collapse-active-hover-shadow;
 					}
 				}
-				i
+				span
```

Comment:
No, it needs the class too, although there aren't any side effects. I did it for our local instance and should have copy and pasted it. Hang on ...